app/grids/ats/profile_tasks_grid.rb

```diff
@@ -49,7 +49,9 @@ class ATS::ProfileTasksGrid
     :assignee,
     html: true,
     preload: { assignee: :account },
-    order: ->(scope) { scope.joins(assignee: :account).group("account.id").order("accounts.name") }
+    order: ->(scope) {
+             scope.left_outer_joins(assignee: :account).group("accounts.id").order("accounts.name")
+           }
```

Comment:
Setting the order by this field removed tasks without assignee from the search.
And fixed typo:  `group("accounts.id")` instead of `group("account.id")`